docs/source/components/stars.ipynb

```diff
@@ -366,7 +366,7 @@
     "print(metal_dist)\n",
     "\n",
     "# Define a constant SFH\n",
-    "sfh = SFH.Constant(duration=100 * Myr)\n",
```

Comment:
I see you're just defining the start and end points now. That makes sense and is more intuitive. You could add the option for any combination of max, min and duration but probably just complicates things for the sake of it.

Reply:
If we only had `Constant` I'd use `duration` and `min_age` (i.e. `max_age = min_age + duration`), with `min_age` by default being 0. However, I think it's better to harmonise with the other models and use `min_age` and `max_age`. `max_age` is more useful in those models because you'd probably naturally set it to the age of the Universe, or some large fraction of it.

I did ponder allowing all three but then you'd have no positional arguments and you'd have to add various logic. 

As annoying as this change is, I think it is better.

Reply:
Yea, on balance I think I agree.